docs/app/pages/components/components-sections/input-controls/typeahead/typeahead.component.html

```diff
@@ -0,0 +1,190 @@
+<div class="row">
+    <div class="col-md-12 has-feedback">
+        <form class="input-typeahead-form">
+            <input [(ngModel)]="input"
+                   [ngModelOptions]="{standalone: true}"
+                   placeholder="Enter Text"
+                   class="form-control"
+                   (click)="dropdownOpen = true"
+                   (keydown)="typeaheadKeyService.handleKey($event, typeahead)"
+                   (keydown.escape)="dropdownOpen = false"
+                   (blur)="dropdownOpen = false">
+
+            <ux-typeahead #typeahead
+                          class="typeahead-example"
+                          [(open)]="dropdownOpen"
+                          [filter]="input"
+                          [options]="loadOptionsFn"
+                          [openOnFilterChange]="true"
+                          [pageSize]="40"
+                          [selectOnEnter]="selectOnEnter"
+                          [selectFirst]="selectFirst"
+                          [dropDirection]="dropDirection"
+                          (optionSelected)="input = $event.option">
+            </ux-typeahead>
+        </form>
+    </div>
+</div>
+
+<div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <ux-accordion>
+            <ux-accordion-panel class="accordion-chevron" heading="Customize Example...">
+                <div class="row uxd-customize-row">
+                    <div class="col-md-4 col-sm-12">
+                        <ux-checkbox [(value)]="selectFirst">selectFirst</ux-checkbox>
+                    </div>
+                    <div class="col-md-4 col-sm-12">
+                        <ux-checkbox [(value)]="selectOnEnter">selectOnEnter</ux-checkbox>
+                    </div>
+                    <div class="col-md-4 col-sm-12">
+                        <label>dropDirection</label>
+                        <ux-radio-button [(value)]="dropDirection" option="down">down</ux-radio-button>
+                        <ux-radio-button [(value)]="dropDirection" option="up">up</ux-radio-button>
+                    </div>
+                </div>
+            </ux-accordion-panel>
+        </ux-accordion>
+    </div>
+</div>
+
+<hr>
+
+<p>The <code>ux-typeahead</code> component can be used to apply a dropdown typeahead list to a text input.</p>
+<p>To match the width of the input control, place both inside a container with a defined width and
+    <code>position: relative</code>.</p>
+<p>It has the following properties:</p>
```

Comment:
Perhaps to keep this better aligned with our other documentation sections this could be something like:

> The following properties can be used to customize the appearance and behavior of the `ux-typeahead` component: